Content.Server/Forensics/Components/ForensicScannerComponent.cs

```diff
@@ -28,6 +28,12 @@ public sealed partial class ForensicScannerComponent : Component
         [ViewVariables(VVAccess.ReadOnly)]
         public List<string> DNAs = new();
 
+        /// <summary>
+        /// Residue that the forensic scanner found from the <see cref="ForensicsComponent"/> on an entity.
+        /// </summary>
+        [ViewVariables(VVAccess.ReadOnly)]
+        public List<string> Residues = new();
+
```

Comment:
Any reason this isn't a datafield?

Reply:
I'd imagine we'd want the scanners to be initially empty, otherwise it might confuse people thinking they scanned something? I'm not quite sure when a datafield would be used here.

Reply:
Gotcha. I'll update it and the existing fields.